src/server/saved_objects/schema/schema.ts

```diff
@@ -31,6 +32,22 @@ export class SavedObjectsSchema {
     this.definition = schemaDefinition;
   }
 
+  public isHiddenType(type: string) {
+    if (this.definition && this.definition.hasOwnProperty(type)) {
+      return !!this.definition[type].hidden;
+    }
+
+    return false;
+  }
+
+  public getAvailableTypes() {
```

Comment:
This method doesn't look used elsewhere, and it's returning entries.

Reply:
Yeah missed in my cleanup of the restricting the saved_objects api to known types.

Reply:
it has been removed.